.gitignore

```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+.env
```

Comment:
Use .env.local instead of .env with .gitignore https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used